.eslintrc

```diff
@@ -1,5 +1,13 @@
 {
   "extends": "lifion",
+  "overrides": [
+    {
+      "files": ["lib/index.test.js"],
+      "rules": {
+        "jest/no-export": ["warn"]
```

Comment:
Unfortunate usage of export.
We should probably address this with fixtures or something.

@eaviles @jennyEckstein

Reply:
Let's follow it up in a separate PR.